pkg/api/validation/cluster_validation_test.go

```diff
@@ -68,7 +68,7 @@ func TestValidateScyllaCluster(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			err := validation.ValidateScyllaCluster(test.obj)
+			err := validation.ValidateScyllaCluster(test.obj).ToAggregate()
```

Comment:
this test should deepequal the fieldErrors

Reply:
fixed